src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -175,7 +162,6 @@ public async Task<bool> GenerateClientAsync(CancellationToken cancellationToken)
 
         try
         {
-            await CleanOutputDirectory(cancellationToken);
```

Comment:
There is a chance this might break someone using KiotaBuilder directly.  I don't know how ApiCurio does their integration and whether they currently rely on that CleanOutputDirectory.  /cc @andreaTP

Reply:
if you are using the builder without the CLI yes. In that case, they'd have to manage cleaning the folder themselves, but they'd also have to already be handling things like the logger too.

Reply:
Thanks for the heads up here! In most of the use cases we are currently defaulting to  a full fresh generation, so, as far as I understand this it's not going to impact us.